include/openssl/ct.h

```diff
@@ -398,7 +402,7 @@ int SCT_LIST_validate(const STACK_OF(SCT) *scts, CT_POLICY_EVAL_CTX *ctx);
  * Returns < 0 on error, >= 0 indicating bytes written (or would have been)
  * on success.
  */
-int i2o_SCT_LIST(const STACK_OF(SCT) *a, unsigned char **pp);
+__owur int i2o_SCT_LIST(const STACK_OF(SCT) *a, unsigned char **pp);
```

Comment:
the i2o_ name is fine, octet string.  reasonable.  Used in i2o_ECPublicKey, e.g.